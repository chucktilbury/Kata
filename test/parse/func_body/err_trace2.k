;-------------------------------------------------------------------------------
;  trace requires a string literal
;-------------------------------------------------------------------------------

function 
	name1 
	() 
	() 
	{
	
	trace()
}
