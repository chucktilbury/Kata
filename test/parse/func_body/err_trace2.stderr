Syntax: /home/chuck/Src/Kata/test/parse/func_body/err_trace2.k: 11: 9: expected a string literal but got )
Syntax: /home/chuck/Src/Kata/test/parse/func_body/err_trace2.k: 13: 1: expected function body element but got end of file
Syntax: /home/chuck/Src/Kata/test/parse/func_body/err_trace2.k: 13: 1: expected function body but got end of file
